packages/react-vapor/src/components/collapsible/CollapsibleInfoBox.tsx

```diff
@@ -20,7 +20,7 @@ export class CollapsibleInfoBox extends React.PureComponent<CollapsibleInfoBoxPr
         return (
             <CollapsibleConnected
                 id={this.props.id}
-                className={classNames(styles.container, 'text-grey-9 mod-rounded-border-2')}
+                className={classNames(styles.container, 'label mod-rounded-border-2')}

```

Comment:
Previously the collaspible infobox had a title that was bolder than its content, its not the case anymore.

### Before
![image](https://user-images.githubusercontent.com/35579930/106811007-b888aa80-663b-11eb-8c78-e3f01338b548.png)

### After 
![image](https://user-images.githubusercontent.com/35579930/106810945-a27aea00-663b-11eb-8679-b0b39b7fece2.png)